specs/protocol/stage-1.md

```diff
@@ -22,7 +22,7 @@
   - [Pause Deputy](#pause-deputy-1)
   - [Architecture Diagram](#architecture-diagram)
 - [Invariants](#invariants)
-  - [iS1-001: A permanent Withdrawal Liveness or Withdrawal Safety failure requires 75% of the Security Council (w/o bugs)](#is1-001-a-permanent-withdrawal-liveness-or-withdrawal-safety-failure-requires-75%25-of-the-security-council-wo-bugs)
+  - [iS1-001](#is1-001)
```

Comment:
Looks like this title was accidentally deleted? We have the titles in place elsewhere

Reply:
If you look at the non-blocking linter issue, you'll see that the title is too long, and the linter does not like it. So I reduced it to the code, and added the remaining details under the title in the respective section.